WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -127,6 +127,29 @@ $venue-map-wrapper-height: 400px;
 }
 
 .competitions-table {
+  font-size: 12px;
```

Comment:
Can we change it from 14px to 12px in a separate, design-only PR? I'd like to think about how this will fit in with the rest of the website, as we have lots of other tables and they're not 12px.